frontend/components/wallet/WalletContext.tsx

```diff
@@ -32,6 +36,7 @@ const defaultImplementation: WalletContextType = {
   addGraveVault: () => {
     // Default addGraveVault implementation
   },
+  connectedChainId: undefined,
```

Comment:
could it be null instead of undefined?

Reply:
As far as I know undefined means a var is declared but not yet assigned a value
null is an explicit assigned value, i.e we're saying "null" would be an acceptable connectedChainId
in this case would undefined be more appropriate? as we have yet to fetch a value (but when we do, it wouldn't be "null")